platform/windows/shared_memory_windows.h

```diff
@@ -0,0 +1,67 @@
+/*************************************************************************/
+/*  shared_memory_posix.h                                            */
```

Comment:
Should be `_windows.h`, and end tag also misaligned.

Reply:
Fixed!

Reply:
Actually this file duplicates `drivers/windows/shared_memory_windows.h`.

Reply:
Oops. You're right. Let me check...

Reply:
Fixed!